apps/docs/src/app/platform/documentation/platform-documentation.component.ts

```diff
@@ -15,15 +15,9 @@ export class PlatformDocumentationComponent {
 
     sideCollapsed: boolean = window.innerWidth < 576;
 
-    guides = [
-        { url: 'platform/home', name: 'Home' },
-        { url: 'platform/new-component', name: 'New Component' }
-    ];
-
-    components = [
-        { url: 'platform/button', name: 'Button' }
+    guides = [{ url: 'platform/home', name: 'Home' }, { url: 'platform/new-component', name: 'New Component' }];
 
-    ];
+    components = [{ url: 'platform/button', name: 'Button' }, { url: 'platform/action-bar', name: 'Action Bar' }];
```

Comment:
Could you relook into alignment because going forward we are going to have more components and we can't place all of them in a single line.